MAINTENANCE.md

```diff
@@ -0,0 +1,15 @@
+<!-- © 2019 Serokell <hi@serokell.io>
+   - © 2019 Lars Jellema <lars.jellema@gmail.com>
```

Comment:
Makes no sense for a file you wrote. Feel free to put your own name or something like "the nixfmt maintainer team" maybe but I'm not sure if that makes sense legally.

Reply:
Yeah I don't really understand what this is about, so I just copied things over to make CI happy ^^

> I'm not sure if that makes sense legally

AFAIK none of this makes sense legally, IIRC copyright notices were a thing only prevalent in american-style jurisdictions, and I think even they lifted that requirement some time ago?

Reply:
Because serokell didn't want to think too much about what is right and wrong for legal stuff like this we use a tool built by the FSFE to decide. I don't really like how it looks but I trust the FSFE to have come up with something that makes sense legally.
More info at https://reuse.software/faq/